controllers/managedfusionoffering_controller.go

```diff
@@ -129,6 +136,14 @@ func (r *ManagedFusionOfferingReconciler) reconcilePhases() (reconcile.Result, e
 					return ctrl.Result{}, fmt.Errorf("failed to remove finalizer from %s secret: %v", managedFusionSecretName, err)
 				}
 			}
+			r.Log.Info(fmt.Sprintf("deleting %s namespace", r.namespace))
+			if err := r.delete(&corev1.Namespace{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: r.namespace,
```

Comment:
- do we already have rbac in-place to delete ns?

Reply:
Yes, we do. Kubebuilder marker is mentioned in managedFusion Controller.